src/components/form-elements/_textarea.scss

```diff
@@ -61,14 +61,18 @@ $includeHtml: false !default;
       }
     }
 
-    &--small {
+    &--clickable {
```

Comment:
TBH I'm not sure about this. Should it really be a part of styleguide? It's not a visual change and it doesn't make sense for most scenarios